pkg/parser/instance/additionalcontainer.go

```diff
@@ -173,14 +174,18 @@ func NewAdditionalContainer(mergedEnv map[string]string, boolevator *boolevator.
 		return nil
 	})
 
-	commandSchema := schema.Script("Container CMD to override.")
+	commandSchema := schema.String("Container CMD to override.")
 	ac.OptionalField(nameable.NewSimpleNameable("command"), commandSchema, func(node *node.Node) error {
-		command, err := node.GetSliceOfNonEmptyStrings()
+		command, err := node.GetExpandedStringValue(mergedEnv)
 		if err != nil {
 			return err
 		}
 
-		ac.proto.Command = command
+		// tokenize the command
+		ac.proto.Command, err = shlex.Split(command)
+		if err != nil {
+			return err
```

Comment:
We should wrap this in a `node.ParserError()` to get a rich error.

Example here: https://github.com/cirruslabs/cirrus-cli/blob/f23edc6fa4cd1e34174b1f119e6dc5e90079ea77/pkg/parser/instance/additionalcontainer.go#L170

Reply:
fixed in 63ab2e0